Application/EdFi.Ods.Api/Helpers/AssemblyLoaderHelper.cs

```diff
@@ -191,7 +203,38 @@ static bool HasPlugin(Assembly assembly)
             }
         }
 
-        private class PluginAssemblyLoadingContext : AssemblyLoadContext
+        private static bool IsExtensionAssembly(AssemblyMetadata assemblyMetadata)
+        {
+            return assemblyMetadata.AssemblyModelType.EqualsIgnoreCase(PluginConventions.Extension);
+        }
+
+        private static bool IsProfileAssembly(AssemblyMetadata assemblyMetadata)
+        {
+            return assemblyMetadata.AssemblyModelType.EqualsIgnoreCase(PluginConventions.Profile);
+        }
+
+        private static AssemblyMetadata LoadAssemblyMetadata(Assembly assembly)
+        {
+            var resourceName = assembly.GetManifestResourceNames()
+                .SingleOrDefault(p => p.EndsWith(AssemblyMetadataSearchString));
+            
+            if (resourceName == null)
+            {
+                throw new FileNotFoundException($"Assembly metadata file '{assembly.GetManifestResourceNames()}' not found for this project.");
+            }
+            var stream = assembly.GetManifestResourceStream(resourceName);

```

Comment:
You need to add blank lines before and after blocks of code. Insert a blank line before `var stream` here.